bitnami/rabbitmq-cluster-operator/templates/_helpers.tpl

```diff
@@ -12,15 +12,23 @@ Return the proper RabbitMQ Messaging Topology Operator fullname
 NOTE: Not using the common function to avoid generating too long names
 */}}
 {{- define "rmqco.msgTopologyOperator.fullname" -}}
-{{- printf "%s-%s" .Release.Name "rabbitmq-messaging-topology-operator"  | trunc 63 | trimSuffix "-" -}}
+{{- if .Values.msgTopologyOperator.fullnameOverride -}}
+    {{- .Values.msgTopologyOperator.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+    {{- printf "%s-%s" .Release.Name "rabbitmq-messaging-topology-operator" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
```

Comment:
I guess we should take into account `.Values.fullnameOverride` parameter if `.Values.msgTopologyOperator.fullnameOverride` is empty, right? I mean, this parameter:

- [values.yaml#L31](https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq-cluster-operator/values.yaml#L31)
Sth like this:

```suggestion
{{- if .Values.msgTopologyOperator.fullnameOverride -}}
    {{- .Values.msgTopologyOperator.fullnameOverride | trunc 63 | trimSuffix "-" -}}
{{- else if .Values.fullnameOverride -}}
    {{- printf "%s-%s" .Values.fullnameOverride "messaging-topology-operator" | trunc 63 | trimSuffix "-" -}}
{{- else -}}
    {{- printf "%s-%s" .Release.Name "rabbitmq-messaging-topology-operator" | trunc 63 | trimSuffix "-" -}}
{{- end -}}
```

Reply:
I also applied this to `rmqco.msgTopologyOperator.webhook.fullname`